src/templates/all/.make/rust.mk

```diff
@@ -134,5 +135,20 @@ rust-grpc-api:
 		pseudomuto/protoc-gen-doc \
 		--doc_opt=json,$(PACKAGE_NAME)-grpc-api.json
 
+rust-coverage: check-cargo-registry rust-docker-pull
+	@echo "$(CYAN)Rebuilding and testing with profiling enabled...$(SGR0)"
+	@docker run \
```

Comment:
This seems to be a copy/paste of the existing cargo_run call with the exception of adding a security-opt.
I think it would be better to add a make var for the security-opt and re-use the existing cargo_run function as it already takes options for the actual cargo command to run

Reply:
https://github.com/Arrow-air/tf-github/pull/74/commits/8df2411f096693836abc9dfc9f4e64b907189c42

added an `ADDITIONAL_OPT` variable for other needs such as this